dataland-frontend/tests/component/components/pages/QualityAssurance.cy.ts

```diff
@@ -1,20 +1,26 @@
-// @ts-nocheck
 import QualityAssurance from '@/components/pages/QualityAssurance.vue';
 import { minimalKeycloakMock } from '@ct/testUtils/Keycloak';
 import { type FixtureData, getPreparedFixture } from '@sharedUtils/Fixtures';
-import { type DataMetaInformation, DataTypeEnum, type PathwaysToParisData } from '@clients/backend';
-import { QaStatus } from '@clients/qaservice';
+import {
+  type CompanyAssociatedDataPathwaysToParisData,
+  type DataMetaInformation,
+  DataTypeEnum,
+  type PathwaysToParisData,
+} from '@clients/backend';
+import { QaStatus, type ReviewQueueResponse } from '@clients/qaservice';
```

Comment:
Diese Zeile wirft in meiner IDE den Fehler -> Vue: Module '"@clients/qaservice"' has no exported member 'ReviewQueueResponse'.

Reply:
Probier lokal bei dir mal aus das e von qaservice in dem import statement zu löschen, einen moment zu warten und es wieder hinzuschreiben. Dann sucht die IDE nochmal neu danach. Das hat in den Fällen wo ich bei mir so etwas beobachtet habe das Problem meistens gelöst. Ansonsten führ nochmal den Tast generateClients aus und schau mal ob der IDE Fehler dann weg geht.

Reply:
Der erste Tipp hat nicht funktioniert, aber 'generateClients' hat das Problem behoben. Danke!